src/components/Icons/Twemoji/Mountain.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+const Mountain = props => (
+  <svg viewBox="0 0 45 45" width={45} height={45} {...props}>
+    <g fill="none">
+      <path
+        fill="#292F33"
+        d="M24 44.346L15.406 14s-1.786.995-3.275 2.14c-.393.303-2.486 3.649-2.907 3.976-2.216 1.719-4.468 3.714-4.77 4.14C3.878 25.074 0 45 0 45l24-.654z"
+      />
+      <path
+        fill="#4B545D"
+        d="M40 44l-6.9-25.937L22.433 4l-6.741 5.507-1.219 1.539c-.51 1.025-2.078 7.017-2.078 7.017L10.828 24.3 3.93 37.504 1 43.726 40 44z"
+      />
+      <path
+        fill="#FFF"
+        d="M4 37s9.973-14.568 10-15.385c.026-.816-1.452-4.615-1.452-4.615l-1.583 6.417L4 37z"
+      />
+      <path
+        fill="#292F33"
+        d="M14.963 42h2.481l2.482-20.784 1.53-2.37 4.44-6.908.116-1.938-4.078 5.877-3.25 4.682-2.962 9.715-.76 2.489-2.221 4.135L10 42h4.963zm14.961-23.246l-2.438 7.782-1.357 4.907 3.464 10.062L34 41.423s-5.39-9.155-5.39-9.98c0-.158.776-3.545.776-3.545l1.357-5.98.349-2.021-1.168-1.143z"
+      />
+      <path
+        fill="#F1F1F1"
+        d="M30.447 19l-2.2-.901-1.896 1.185-1.643-1.44L22 19.094 26.428 12l4.019 7zM34 29l-2.118-1.881-1.972 1.271 1.354-6.142 1.778 3.093L34 29z"
+      />
+      <path
+        fill="#FFF"
+        d="M28 26.648l-1.38 4.902-.316-3.186L28 26.648zm-12.74 6.22L13 37l.446-3.205s.86-.762 1.183-1.2c.322-.44 1.404-2.213 1.404-2.213l-.773 2.487zm4.05-14.5L22.351 16l-3.305 4.677.262-2.31z"
+      />
+      <path
+        fill="#292F33"
+        d="M45 45s-.49-5.134-1.091-9.457c-.472-3.395-3.523-8.415-5.238-9.499l-1.577-4.72-2.585-6.813-2.885-3.056-2.356-2.17-.645-.83L22 3l2.925 6.685.67 2.23 4.147 6.895.843 3.2 1.838 3.046.989 3.606 1.366 4.99-2.556 8.678L45 45z"
+      />
+      <path
+        fill="#5C903F"
+        d="M42.135 40.5c-1.681 0-6.197.71-6.822.71-2.5 0-5.313-2.21-6.563-2.21s-4.492 2.21-6.992 2.21c-2.11 0-3.008-1.025-5.9-1.025-.934 0-5.12 1.1-6.014.986-2.279-.288-3.45-1.224-4.024-1.224C2.5 39.947 0 45 0 45h45s-1.094-4.5-2.865-4.5z"
+      />
+      <path
+        fill="#F1F1F1"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 spaces but found 8. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)